src/dialog/dialog.jsx

```diff
@@ -383,19 +384,6 @@ export default class Dialog extends Component {
 
         const inner = this.renderInner(canCloseByCloseClick);
 
-        return (
-            <Overlay {...newOverlayProps}>
-                {useCSS ? (
-                    <div
-                        className={`${prefix}dialog-container`}
-                        dir={rtl ? 'rtl' : undefined}
-                    >
-                        {inner}
-                    </div>
-                ) : (
-                    inner
-                )}
-            </Overlay>
-        );
+        return <Overlay {...newOverlayProps}>{inner}</Overlay>;
```

Comment:
the dom structor of full screen dialog changed, this should be a minor change, switch target branch to feat/1.17.0

Reply:
Is this still necessary as master is now 1.17.0?